Various/Boekhoudprogramma.bc3

```diff
@@ -551,13 +551,15 @@
 6590 AH=AH-1
 6595 GOSUB 1282
 6600 RETURN
-10000 SR$="":IF CT<=0 THEN RETURN ELSE OS#=CDBL(ABS(SR#))
-10010 IF CN<=0 THEN OI#=INT(OS#+.5):OF#=0:GOTO 10030 ELSE OI#=INT(OS#+.5*10^-CN):OF#=INT(((OS#-OI#)*10^CN)+.5)
-10020 OFIP#=OF#:GOSUB 10060:IF OX$="" GOTO 10050 ELSE SR$="."+STRING$(CN-LEN(OX$),48)+OX$
-10030 OFIP#=OI#:GOSUB 10060:IF OX$="" GOTO 10050 ELSE SR$=OX$+SR$:IF(SGN(SR#)<0)AND(NOT((OI#=0)AND(OF#=0)))THEN SR$="-"+SR$
+10000 SR$="":IF CT<=0 THEN RETURN: OS=CDBL(ABS(SR))

```

Comment:
I doubt this is the right thing to do - while `ELSE` may not be allowed in Basicode (not in 2, would need to check in BC3), just removing it changes the logic. In the original the statements after `ELSE` would be intended to be executed if the premise of the `IF` is false, but in the modified version they will never be executed as they follow an unconditional jump.

Reply:
In general in this repo I've stored programs as I've found them to document the original, unfortunately not all will necessarily run in the interpreter. This may be because of bugs in the interpreter, or because the program author did not strictly follow the Basicode spec (most of the time authors would have tested only on their own computer, not on all platforms supported by Basicode), or because the Basiode spec was unclear about the matter. Users would often have had to amend program code to make it run on their computer. Perhaps the thing we should do is create a branch of code that contains fixes to make it run where the code clearly breaks the spec.

Reply:
Ah yes, I see that now. I didn't verified the modified code too much, it was enough for me that it interpreted.

It seems that ELSE could be forbidden in all(?) versions of BASICODE (second link).
https://archive.org/details/BASICODE2Manual/page/n9/mode/2up?view=theater
http://www.archaeocosmology.org/eng/BASICODE3sub.htm

Related observation: manual to ver. 2 indicates that IF should go with THEN, but at the same time it doesn't clearly forbids the lack of it (unlike the ELSE or !/#/% types after the variable).

Reply:
I've created [Boekhoudprogramma.bc2](https://github.com/Geist-zz/basicode/blob/master/Various/Boekhoudprogramma.bc2) on my repository. Modified it to run (to a point) + Google translated to English prompts & comments + corrected ELSEs (or so I think).
There are still couple of unsupported STRING$ left.

I gave up at the point of trying to add (in the program) "changes" to the "main group" (last "from" in prompt could indicate that I ended up on the wrong path?).
It would be good if the online interpreter had some debug options (ex. stepping or tracing to browser console)...

Creation of "main group":
tst
2022
22
1
b
1
test
z/i/k/p (one of these)
j

Trying to add "changes" to that group:
1
a
25
6
100
1 (just redraws the screen from what I see)